include/fmt/format.h

```diff
@@ -2660,7 +2660,9 @@ class format_string_checker {
   // Format specifier parsing function.
   using parse_func = const Char* (*)(parse_context_type&);
 
-  unsigned arg_id_;
+  enum { ARG_ID_SENTINEL = -1 };
```

Comment:
This is not needed, negative value is a clear enough indication of an invalid index.

Reply:
OK